webapp/certified/views.py

```diff
@@ -114,6 +116,10 @@ def certified_hardware_details(canonical_id, release):
 
 def certified_model_details(canonical_id):
 
+    models = api.certified_models(canonical_id=canonical_id)["objects"]
```

Comment:
I missed this at first. You don't actually need to call the certifiedmodels api. The `category` info is available from certifiedmodeldetails now. A while back I saw that it should be in there so that people wouldn't need to make two api calls to get category information.